functions.php

```diff
@@ -904,12 +904,12 @@ public function override_img_caption_shortcode( $output, $attr, $content ) {
 
 		return $output;
 	}
-
 }
 
 new P4_Master_Site( [
 	'P4_Custom_Taxonomy',
 	'P4_Taxonomy_Image',
 	'P4_Settings',
 	'P4_Control_Panel',
+	'P4_Post_Report_Controller',
```

Comment:
@kirdia could we avoid loading this controller here and instead create it within the posts_report.php so that this functionality actually gets loaded only in the desired backend page instead.

Reply:
ps. at some point, we could do this for the other classes as well (taxonomy_image, settings, control_panel, etc...)

Reply:
Unfortunately it can't be placed in posts_report.php. There are some wordpress hooks that need to run early on the theme load process.